models/QtiRunnerInitDataBuilder.php

```diff
@@ -303,4 +314,28 @@ private function getResponseCountsFromState(array $itemState): int
 
         return $responsesCount;
     }
+
+    /**
+     * @throws MissingOutcomeDeclarationMaxScore
+     */
+    private function getMaxScore($outcomes): float
+    {
+        foreach ($outcomes as $outcome) {
+            if ($outcome[self::OUTCOME_DECLARATION_IDENTIFIER] === self::OUTCOME_DECLARATION_MAXSCORE) {
+                return (float) $outcome[self::OUTCOME_DECLARATION_DEFAULT_VALUE];
+            }
+        }
+
+        throw new MissingOutcomeDeclarationMaxScore();
```

Comment:
The following seems like a more compatible approach to me.
```suggestion
        return null;
```

Return type of the method will also have to be changed then of course.